### PR TITLE
re-add GitPod w --no-sandbox headless chrome option

### PR DESCRIPTION
**Pull request by jywarren**
Follow-up to un-revert Gitpod in #716

Follow-up to #680

### CODE REVIEW

**Comment by jywarren** [comment]
Hmm, Gitpod outputs:

```

Running "karma:development:start" (karma) task

START:
15 09 2020 21:28:14.938:INFO [karma-server]: Karma v5.1.1 server started at http://localhost:9876/
15 09 2020 21:28:14.941:INFO [launcher]: Launching browsers ChromeHeadless with concurrency unlimited
15 09 2020 21:28:14.949:INFO [launcher]: Starting browser ChromeHeadless
15 09 2020 21:28:14.958:ERROR [launcher]: No binary for ChromeHeadless browser on your platform.
  Please, set "CHROME_BIN" env variable.

Finished in 0 secs / 0 secs @ 21:28:14 GMT+0000 (Coordinated Universal Time)

Warning: Task "karma:development:start" failed. Use --force to continue.

Aborted due to warnings.
npm ERR! code ELIFECYCLE
npm ERR! errno 3
npm ERR! leaflet-distortableimage@0.21.8 build: `cross-env NODE_ENV=production grunt build`
npm ERR! Exit status 3
npm ERR! 
npm ERR! Failed at the leaflet-distortableimage@0.21.8 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/gitpod/.npm/_logs/2020-09-15T21_28_15_091Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 3
npm ERR! leaflet-distortableimage@0.21.8 setup: `npm i && npm run build`
npm ERR! Exit status 3
npm ERR! 
npm ERR! Failed at the leaflet-distortableimage@0.21.8 setup script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/gitpod/.npm/_logs/2020-09-15T21_28_15_136Z-debug.log
```

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+tasks:
+  - init: npm i && npm run setup
+    command: npm run serve:dev
```
Comment (jywarren) [inline .gitpod.yml:3]: I'm adding `NODE_ENV=production node_modules/grunt/bin/grunt webpack` in front of this, so we get all deps built for GitPod!

**Comment by jywarren** [comment]
OK, the `karma:development:start` won't run in GitPod, but that's OK -- we can fix that either here or in follow-up. 

cc @sashadev-sky -- i think we would just need to set the `CHROME_BIN` variable!

**Comment by jywarren** [comment]
https://toedter.com/2018/10/27/gitpod-web-ide-for-github/

**Comment by jywarren** [comment]
Looks good, i think we got it, including the test run!

**Comment by jywarren** [comment]
Yeahhhh!!!

image

**Comment by sashadev-sky** [comment]
🚀 thank you for taking care of this jeff! Sorry I removed it - I had a hard time deciding if it would be better to drop support or use a flag that is a workaround

**Pull request merged**